rules/windows/file/file_event/file_event_win_office_macro_files_from_susp_process.yml

```diff
@@ -0,0 +1,51 @@
+title: Office Macro File Creation From Suspicious Process
+id: b1c50487-1967-4315-a026-6491686d860e
+status: experimental
+description: Detects the creation of a office macro file from a a suspicious process
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/f339e7da7d05f6057fdfcdd3742bfcf365fee2a9/atomics/T1566.001/T1566.001.md
+    - https://docs.microsoft.com/en-us/deployoffice/compat/office-file-format-reference
+author: frack113, Nasreddine Bencherchali (Nextron Systems)
+date: 2022/01/23
+modified: 2023/02/22
+tags:
+    - attack.initial_access
+    - attack.t1566.001
+logsource:
+    category: file_event
+    product: windows
+    definition: 'Requirements: The "ParentImage" field is not available by default on EID 11 of Sysmon logs. To be able to use this rule to the full extent you need to enriche the log with additional ParentImage data'
+detection:
+    selection_cmd:
+        - Image|endswith:
+            - '\cmd.exe'
```

Comment:
for cmd+pwsh: It would trigger if you copy a file using the CLI, doesn't it? --> wouldn't include them in a high rule. 
The other images are fine